core/src/main/java/bisq/core/user/Preferences.java

```diff
@@ -122,6 +122,8 @@
     @Getter
     private final BooleanProperty useAnimationsProperty = new SimpleBooleanProperty(prefPayload.isUseAnimations());
     @Getter
+    private final BooleanProperty useDarkModeProperty = new SimpleBooleanProperty(prefPayload.isUseDarkMode());
```

Comment:
See my comment on the the data type of the ui theme above.

Reply:
This code is outdated, as per your earlier review #2414 was changed from using a `bool` to a `UiTheme` object which allows for any number of themes: https://github.com/peterzen/bisq/blob/f35a0d58b2aef137b9b4fe9f37f27e78ce00f20c/common/src/main/proto/pb.proto#L1855